src/org/loklak/data/DAO.java

```diff
@@ -978,6 +987,25 @@ public SearchLocalMessages(final String q, final Timeline.Order order_field, fin
             }
             this.aggregations = query.aggregations;
         }
+
+        public SearchLocalMessages (
+                final String q,
+                final Timeline.Order order_field,
+                final int timezoneOffset,
+                final int resultCount,
+                final int aggregationLimit,
+                final String... aggregationFields
+        ) {
+            this(
+                q,
+                order_field,
+                timezoneOffset,
+                resultCount,
+                aggregationLimit,
+                new ArrayList<>(),
+                aggregationFields
+            );
+        }
```

Comment:
It is indeed more readable, but it can be like this too - 
```java
this(q, orderField, timezoneOffset, resultCount, aggregationLimit,
    new ArrayList<>(), aggregationFields);
```

Again, it is all about personal taste. So, according to me, it is better to stick to the current conventions of the project.